src/components/Pages/EventList/EventListEntry.js

```
@@ -8,100 +8,127 @@ import { useMobileQuery } from 'js/helpers/reactMediaQueries.js';
+  let locationName = null;
   if (location) {
-    locationName = (location.locationType === 'city_location')
-      ? location.cityLocation.title
-      : location.remoteLocation.name;
+    locationName =
+      location.locationType === 'city_location'
+        ? location.cityLocation.title
+        : location.remoteLocation.name;
   }
 
   // Joplin lets a user mark an event as free but also include costs.
   // Or not mark as free, but not include fees. Ideally Id like to just check if eventIsFree
   // let cost = (eventIsFree) ? `${intl.formatMessage(i18n.free)}` : feesRange;
   // In the interim, this will have to do. b/c if a user doesnt enter fees, it should say free -chia
-  let cost = (feesRange.length === 0) ? `${intl.formatMessage(i18n.free)}` : feesRange;
-  let registration = registrationUrl.length ? `• ${intl.formatMessage(i18n.registrationReq)}`: '';
+  let cost =
+    feesRange.length === 0 ? `${intl.formatMessage(i18n.free)}` : feesRange;
+  let registration = registrationUrl.length
+    ? `• ${intl.formatMessage(i18n.registrationReq)}`
+    : '';
 
-  if (cost.slice(0,2) === '$0') {
-    cost = `${intl.formatMessage(i18n.free)}${cost.slice(2)}`
+  if (cost.slice(0, 2) === '$0') {
+    cost = `${intl.formatMessage(i18n.free)}${cost.slice(2)}`;
   }
 
   return (
     <div
-      className={classNames('coa-EventListPage__EntryDetails', 
-          {'coa-EventListPage__EntryDetails--homepage': homepage} )}> 
-      { 
-        canceled 
-        && <div className="coa-EventListPage__Canceled">
-            {intl.formatMessage(i18n.canceled)}
-           </div> 
-      }
-      <div className="coa-EventListPage__Time">{ `${momentDay} • ${eventTime}`}</div>
-      <div className="coa-EventListPage__Title"><a href={eventUrl}>{title}</a></div> 
+      className={classNames('coa-EventListPage__EntryDetails', {
+        'coa-EventListPage__EntryDetails--homepage': homepage,
+        'coa-EventListPage__EntryDetails--relatedLocation': relatedLocation,
```

most everything is jsprettier except passing through booleans to toggle styles